modules/ROOT/pages/indexes/search-performance-indexes/drop-indexes.adoc

```diff
@@ -0,0 +1,145 @@
+:description: This page explains how to drop indexes used for search performance.
+= Drop indexes
+
+An index can be dropped (removed) using the name with the `DROP INDEX index_name` command.
+This command can drop indexes of any type, except those backing constraints.
+The name of the index can be found using the xref:indexes/search-performance-indexes/list-indexes.adoc[`SHOW INDEXES` command], given in the output column `name`.
+
+////
+The below indexes are created in schema/indexes/create-indexes.adoc
+[source, cypher, role=test-setup]
+----
+CREATE INDEX node_range_index_name FOR (n:Person) ON (n.surname)
+CREATE INDEX rel_range_index_name FOR ()-[r:KNOWS]-() ON (r.since)
+CREATE INDEX composite_range_node_index_name FOR (n:Person) ON (n.age, n.country)
+CREATE INDEX composite_range_rel_index_name FOR ()-[r:PURCHASED]-() ON (r.date, r.amount)
+CREATE INDEX $name FOR (n:Person) ON (n.firstname)
+CREATE INDEX node_range_index_name IF NOT EXISTS FOR (n:Person) ON (n.surname)
+CREATE TEXT INDEX node_text_index_nickname FOR (n:Person) ON (n.nickname)
+CREATE TEXT INDEX rel_text_index_name FOR ()-[r:KNOWS]-() ON (r.interest)
+CREATE TEXT INDEX $name FOR (n:Person) ON (n.favoriteColor)
+CREATE TEXT INDEX node_index_name IF NOT EXISTS FOR (n:Person) ON (n.nickname)
+CREATE POINT INDEX node_point_index_name FOR (n:Person) ON (n.sublocation)
+CREATE POINT INDEX rel_point_index_name FOR ()-[r:STREET]-() ON (r.intersection)
+CREATE POINT INDEX $name FOR ()-[r:STREET]-() ON (r.coordinate)
+CREATE POINT INDEX node_point_index IF NOT EXISTS FOR (n:Person) ON (n.sublocation)
+CREATE POINT INDEX point_index_with_config FOR (n:Label) ON (n.prop2) OPTIONS {
+  indexConfig: {
+    `spatial.cartesian.min`: [-100.0, -100.0],
+    `spatial.cartesian.max`: [100.0, 100.0]
+  }
+}
+CREATE LOOKUP INDEX node_label_lookup_index FOR (n) ON EACH labels(n)
+CREATE LOOKUP INDEX rel_type_lookup_index FOR ()-[r]-() ON EACH type(r)
+CREATE LOOKUP INDEX node_label_lookup IF NOT EXISTS FOR (n) ON EACH labels(n)
+CREATE INDEX example_index FOR (n:Book) ON (n.title)
+CREATE INDEX bookTitleIndex FOR (book:Book) ON (book.title)
+CREATE TEXT INDEX indexOnBooks FOR (b:Label1) ON (b.prop1)
+CREATE INDEX indexOnBooks FOR (book:Book) ON (book.numberOfPages)
+CREATE CONSTRAINT uniqueBookIsbn FOR (book:Book) REQUIRE (book.isbn) IS UNIQUE
+CREATE INDEX bookIsbnIndex FOR (book:Book) ON (book.isbn)
+CREATE CONSTRAINT bookRecommendations FOR (book:Book) REQUIRE (book.recommend) IS NOT NULL
+CREATE INDEX bookRecommendations FOR (book:Book) ON (book.recommendations)
```

Comment:
These come from failing examples so the failing parts of them should not be part of this setup (only the things that actually got created should be here)
```suggestion
CREATE INDEX example_index FOR (n:Book) ON (n.title)
CREATE TEXT INDEX indexOnBooks FOR (b:Label1) ON (b.prop1)
CREATE CONSTRAINT uniqueBookIsbn FOR (book:Book) REQUIRE (book.isbn) IS UNIQUE
CREATE CONSTRAINT bookRecommendations FOR (book:Book) REQUIRE (book.recommend) IS NOT NULL
```